apps/playground/package.json

```diff
@@ -24,6 +24,6 @@
     "@types/react": "^18.0.22",
     "@types/react-dom": "^18.0.7",
     "eslint-config-custom": "*",
-    "typescript": "^4.5.3"
+    "typescript": "^5.3.3"
```

Comment:
Upgraded it while I was in the weeds of TS. Now that I'm through it, don’t think we absolutely _have to_, but decided to keep it anyway.